airgun/entities/usergroup.py

```diff
@@ -0,0 +1,64 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.usergroup import (UserGroupCreateView,
+                                    UserGroupDetailsView, UserGroupsView)
+
+
+class UserGroupEntity(BaseEntity):
+
+    def create(self, values):
+        view = self.navigate_to(self, 'New')
+        view.fill(values)
+        view.submit.click()
+
+    def search(self, value):
+        view = self.navigate_to(self, 'All')
+        return view.search(value)
+
+    def read(self, entity_name):
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        return view.read()
+
+    def update(self, entity_name, values):
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.fill(values)
+        view.submit.click()
+
+    def delete(self, entity_name):
+        view = self.navigate_to(self, 'All')
+        view.search(entity_name)
+        view.table.row(name=entity_name)['Actions'].widget.click(
+                handle_alert=True)
+
+
+@navigator.register(UserGroupEntity, 'All')
+class ShowAllUserGroups(NavigateStep):
+    VIEW = UserGroupsView
+
+    def step(self, *args, **kwargs):
+        self.view.menu.select('Administer', 'User Groups')
+
+
+@navigator.register(UserGroupEntity, 'New')
+class AddNewUserGroup(NavigateStep):
+    VIEW = UserGroupCreateView
+
+    prerequisite = NavigateToSibling('All')
+
+    def step(self, *args, **kwargs):
+        self.parent.new.click()
+
+
+@navigator.register(UserGroupEntity, 'Edit')
+class EditUser(NavigateStep):
```

Comment:
`EditUser` ?

Reply:
`EditUserGroup` of course. I used `entities/user.py` as example.